desk/app/expose.hoon

```diff
@@ -236,7 +236,7 @@
     ::
         %fact
       ?.  =(%channel-response-1 p.cage.sign)  [~ this]
-      =+  !<(r-channels:d q.cage.sign)
+      =+  !<(r-channels:v7:old:d q.cage.sign)
```

Comment:
This is questionable, because this type doesn't match that of the `%channel-response-1` mark. Is that an oversight?

Reply:
And maybe this isn't even right, because `response-2` _has_ the `v7` type...

Reply:
Given that %channels doesn't actually emit `%response-1` facts, I think this is incorrect. (I can't recall the reason  for `%response-1` being unused); the marks that are in use:
```
%channel-response -> $r-channels-simple-post:v7:old:c
%channel-response-2 -> $r-channels:v7:old:c
%channel-response-3 -> $r-channels:c
```

On the path `/v1` to which %expose subscribes to %channel-response-2 facts are sent.

Reply:
Btw, in connection with this, I have been thinking about making marks behave like type labels, which they in fact are, and let them carry a version number of the underlying type. So we would have
`%channel-response-7` instead of `%channel-response-2` for instance.

I don't think incrementing marks like we do currently brings much benefit.